client/Router.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import toRegex from 'path-to-regexp';
```

Comment:
Why am I surprised this is a package that exists.

Reply:
Surprisingly useful :)